tests/hawc/apps/hawc_admin/test_api.py

```diff
@@ -81,3 +84,24 @@ def test_assessment_values(self):
         assert resp.status_code == 200
         df = pd.read_json(StringIO(resp.content.decode()))
         assert df.shape == (3, 33)
+
+
+@pytest.mark.django_db
+class TestAdminActions:
+    def test_delete_orphan_tags(self, db_keys):
```

Comment:
Is there a quick way to produce an orphan tag? I tried just deleting a parent (e.g. Inclusion) but that seemed to clean up the children too

Reply:
you could probably make one in SQL using the django run sql command and set the depth to 2 and the path to something like 99990001.  You can add that as a todo and mark as xfail and we can fix later, i wouldn't spend a lot more time on it https://docs.pytest.org/en/8.1.x/how-to/skipping.html#xfail-mark-test-functions-as-expected-to-fail

Reply:
I ended up removing this admin action since it wasn't used as much as I thought we needed it; and added a different test to check the clean_orphans implemented in `hawc.apps.common.models.AssessmentRootMixin`